### PR TITLE
Avoid path map solely based on file name

### PR DESCRIPTION
**Pull request by keshav-space**
- fixes https://github.com/nexB/scancode.io/issues/872

### CODE REVIEW

File: scanpipe/pipes/d2d.py [hunk 1]
```diff
@@ -421,7 +421,10 @@ def _map_path_resource(
     # Only create relations when the number of matches if inferior or equal to
     # the current number of path segment matched.
     if len(match.resource_ids) > match.matched_path_length:
-        to_resource.update(status=flag.TOO_MANY_MAPS)
+        # Single matched path segment/file name don't
```
Comment (pombredanne) [inline scanpipe/pipes/d2d.py:424]: IMHO we should go further and not store any mapping at all in this case

Comment (keshav-space) [reply scanpipe/pipes/d2d.py:424]: Ack, So we don't want to path map any files solely based on the file name.

**Comment by tdruez** [comment]
@keshav-space is this ready to be merged?

**Comment by keshav-space** [comment]
> @keshav-space is this ready to be merged?

@tdruez this is ready for merge!

**Pull request merged**